CorsixTH/Lua/languages/russian.lua

```diff
@@ -61,10 +64,15 @@ new_game_window = {
   player_name = "Имя Игрока",
   start = "Старт",
 }
+
+custom_campaign_window = {
+  caption = "Дополнительные кампании",
+  start_selected_campaign = "Начать кампанию",
+}
 menu_options_warmth_colors = {
-  choice_2 = "  СИНИЙ ЗЕЛЕНЫЙ КРАСНЫЙ  ",
-  choice_1 = "  КРАСНЫЙ  ",
-  choice_3 = "  ЖЕЛТЫЙ ОРАНЖЕВЫЙ КРАСНЫЙ  ",
+  choice_2 = "  Синий, зеленый, красный ",
+  choice_1 = "  Красный  ",
+  choice_3 = "  Желтый, оранжевый, красный  ",
```

Comment:
I think some menu items are intentionally capitalised, but I don't have the documentation to support it. Do we all caps in other languages?

Reply:
> I think some menu items are intentionally capitalised, but I don't have the documentation to support it. Do we all caps in other languages?

Yes in other languages there is caps, but due to option menu was already not using caps in russian.lua, i decided to keep it like that. Also i would notice that Arial (which is a default font for unicode languages) looks... well, not good while being capitalised in a menu.

In this PR
![image](https://github.com/CorsixTH/CorsixTH/assets/91145555/d8771e4f-5c7e-476d-8db4-15a7df5d419d)
In CorsixTH v0.67
![image](https://github.com/CorsixTH/CorsixTH/assets/91145555/7d0b8168-b306-4fd9-8db2-68931e0dcf08)

Reply:
Could you try testing out GoNotoKurrent-Bold.ttf as your Unicode font linked in #2398 ? It'd be interesting how readable russian becomes.

Reply:
> Could you try testing out GoNotoKurrent-Bold.ttf as your Unicode font linked in #2398 ? It'd be interesting how readable russian becomes.

It's alright but the text is being a little scattered and sometimes goes beyond of dialog borders

![image](https://github.com/CorsixTH/CorsixTH/assets/91145555/6b08e36f-f534-482e-b8a4-41d84c809867)

Reply:
That's usually expected with a font change annoyingly. Does GoNotoKurrent-Regular look better (also in that PR I think)?